models/marts/core/int_all_graph_resources.sql

```diff
@@ -61,10 +61,10 @@ joined as (
         end as model_type_prefix,
         case 
             when unioned_with_calc.resource_type in ('test', 'source', 'metric', 'exposure', 'seed') then null
-            when {{ dbt.position('naming_convention_folders.folder_name_value','unioned_with_calc.directory_path') }} = 0 then null
+            when {{ dbt.position("concat('/',naming_convention_folders.folder_name_value,'/')",'unioned_with_calc.directory_path') }} = 0 then null
```

Comment:
`concat()` doesn't work with more than 2 arguments on Redshift, hence why the failing CI check. 
You could most likely use [`dbt.concat`](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros#concat) instead.